src/PrestaShopBundle/Form/Admin/Type/DatePickerType.php

```diff
@@ -26,8 +26,10 @@
 
 namespace PrestaShopBundle\Form\Admin\Type;
 
+use PrestaShopBundle\Form\DataTransformer\ArabicToLatinDigitDataTransformer;
```

Comment:
Shouldn't this be a DataTransformerInterface?
```suggestion
use Symfony\Component\Form\DataTransformerInterface;
```